app/views/software_records/_form_server_environment.html.erb

```diff
@@ -18,11 +18,44 @@
               <%= form.text_area :production_support_servers, :class => "form-control" %>
             </div>
 
+            <div class="form-group">
+              <%= form.label :qa_support_servers %>
+              <%= form.text_area :qa_support_servers, :class => "form-control" %>
+            </div>
+
+            <div class="form-group">
+              <%= form.label :dev_support_servers %>
+              <%= form.text_area :dev_support_servers, :class => "form-control" %>
+            </div>
+
             <div class="form-group">
               <%= form.label "last_record_change" %>
                <%= form.date_field :last_record_change, as: :date, value: form.object.try(:strftime,"%m/%d/%Y"), class: 'form-control' %>
             </div>
 
+            <div class="form-group">
+              <%= form.label "date_cert_expires" %>
+               <%= form.date_field :date_cert_expires, as: :date, value: form.object.try(:strftime,"%m/%d/%Y"), class: 'form-control' %>
+            </div>
+
+            <% if params[:id].nil? %>
+              <div class="form-group">
+                <%= form.label :monitor_certificates, 'Track certificates?' %><br/>
+                <%= form.label :monitor_certificates, "Yes", :value => "Yes"  %>
+                <%= form.radio_button :monitor_certificates, "Yes" %>
+                <%= form.label :monitor_certificates, "No", :value => "No" %>
+                <%= form.radio_button :monitor_certificates, "No", :checked => true %>
+              </div>
+            <% else %>
+              <div class="form-group">
+                <%= form.label :monitor_certificates, 'Track uptime?' %><br/>
```

Comment:
Form label text here should be "Track Certificates". I think this came over from copy/paste?
"Track Uptime" label is duplicated. 

![Screen Shot 2023-07-17 at 3 12 35 PM](https://github.com/uclibs/application_portfolio/assets/1069588/228b8836-fcbf-424e-a680-6d4c2ae1866c)